app/src/cards/DisparityBarChartCard.tsx

```diff
@@ -2,85 +2,117 @@ import React, { useState } from "react";
 import DisparityBarChart from "../charts/DisparityBarChart";
 import styles from "./Card.module.scss";
 import { Alert } from "@material-ui/lab";
-import { Row } from "../data/DatasetTypes";
 import { CardContent } from "@material-ui/core";
 import ToggleButton from "@material-ui/lab/ToggleButton";
 import ToggleButtonGroup from "@material-ui/lab/ToggleButtonGroup";
 import SimpleHorizontalBarChart from "../charts/SimpleHorizontalBarChart";
-import { VariableId } from "../data/variableProviders";
 import { Fips } from "../utils/madlib/Fips";
 import {
   BreakdownVar,
   BREAKDOWN_VAR_DISPLAY_NAMES,
   MetricToggle,
+  shareOf,
   METRIC_FULL_NAMES,
   METRIC_SHORT_NAMES,
 } from "../utils/madlib/DisplayNames";
+import useDatasetStore from "../data/useDatasetStore";
+import { Breakdowns } from "../data/Breakdowns";
+import { getDependentDatasets, VariableId } from "../data/variableProviders";
+import VariableQuery from "../data/VariableQuery";
 
 import CardWrapper from "./CardWrapper";
 
 export type ChartToggle = "percents" | "ratio";
 
 function DisparityBarChartCard(props: {
-  dataset?: Row[];
-  datasetIds: string[];
   breakdownVar: BreakdownVar;
   metricId: MetricToggle;
   fips: Fips;
 }) {
   const [chartToggle, setChartToggle] = useState<ChartToggle>("percents");
 
+  const datasetStore = useDatasetStore();
+
+  // TODO need to handle race categories standard vs non-standard for covid vs
+  // other demographic.
+  const geoFilteredBreakdowns = Breakdowns.forFips(props.fips).andRace(true);
```

Comment:
Now that these mostly just have one query, probably fine to drop the overly-specific names and just change them to `query`